### ISSUE TITLE
Backbone of yolo v7

### ISSUE THREAD

**Issue by chey0313** [jinfagang/yolov7#207]
Hello, I appreciate your novel work.
I have a question about your implementation.
From official yolo v7 github, there are no public backbone network application around yolo v7 variants.
Therefore, I can't find the certain name of yolov7 backbone or head network.
Then, Could you give me an exact combination for each yolov7 variant from your implementation?
For example,  darknet53 + yolofpn + yoloxhead = yolo_v7_xxx.

Best regards.

**Comment by lucasjinreal** [comment]
No, I was unable to reproduce yolov7 performance, you can try yolox though.
